### PR TITLE
Bundle dust-api-client types

### PR DESCRIPTION
**Pull request by Fraggle**
## Description

Take 2 on https://github.com/dust-tt/dust/pull/8111

Changes:
- Fix the `use_workspace_credentials` handling to only accept "true" (and not any truthy value)
- Remove the unrelated .husky change.

## Risk

The original version created an incident.

## Deploy Plan

Test on front edge, deploy prod.

### CODE REVIEW

File: front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/* eslint jsdoc/no-missing-syntax: 0 */ //
+// Disabling jsdoc rule, as we're not yet documentating dust apps endpoints under vaults.
+// We still document the legacy endpoint, which does the same thing.
+// Note: for now, an API key only has access to the global vault.
```
Comment (flvndvd) [inline front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts:4]: ✂️

File: front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/[runId]/index.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/* eslint jsdoc/no-missing-syntax: 0 */ //
```
Comment (flvndvd) [inline front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/[runId]/index.ts:1]: ✂️

File: front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/index.ts [hunk 3]
```diff
@@ -1,4 +1,9 @@
-import type { AppType, WithAPIErrorResponse } from "@dust-tt/types";
+/* eslint jsdoc/no-missing-syntax: 0 */ //
```
Comment (flvndvd) [inline front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/index.ts:1]: ✂️

**Comment by flvndvd** [comment]
> Fix the use_workspace_credentials handling to only accept "true" (and not any truthy value)

Can you add a comment on where the fix is in the code?

File: front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts [hunk 4]
```diff
@@ -222,7 +223,8 @@ async function handler(
 
   // This variable is used in the context of the DustAppRun action to use the workspace credentials
   // instead of our managed credentials when running an app with a system API key.
-  const useWorkspaceCredentials = !!req.query["use_workspace_credentials"];
+  const useWorkspaceCredentials =
+    req.query["use_workspace_credentials"] === "true";
```
Comment (Fraggle) [inline front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts:227]: @flvndvd the fix is here.

File: front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/[runId]/index.ts (hunk 2 above)
Comment (Fraggle) [reply front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/[runId]/index.ts:1]: got confused by the diff.

Comment (flvndvd) [reply front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/[runId]/index.ts]: Still need to remove, right?

File: front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts (hunk 1 above)
Comment (flvndvd) [reply front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts]: Same here.

**Review by flvndvd — APPROVED** [review]
LGTM

**Pull request merged**